File: csharp/gabaritos/gabarito10.cs
using System;
using System.Collections.Generic;
using System.Linq;

class Program
{
    static void Main()
    {
        List<int> numeros = new List<int> { 10, 20, 30, 40, 50 };
        double soma = numeros.Sum();
        double media = soma / numeros.Count;
        Console.WriteLine($"A média é: {media}");
    }
}

File: cpp/gabaritos/gabarito10.cpp
#include <iostream>
#include <vector>
#include <numeric>

int main() {
    std::vector<int> nums = {10, 20, 30, 40, 50};
    int soma = std::accumulate(nums.begin(), nums.end(), 0);
    double media = static_cast<double>(soma) / nums.size();
    std::cout << "A média é: " << media << std::endl;
    return 0;
}